### PR TITLE
update Scala to 2.13.5

### PR DESCRIPTION
**Pull request by novakov-alexey-zz**

### CODE REVIEW

**Comment by novakov-alexey-zz** [comment]
new mill-scalafix version needs to be released that brings latest semanticdb library 4.4.10, currently it is 4.4.0:

https://github.com/joan38/mill-scalafix/blob/master/mill-scalafix/src/com/goyeau/mill/scalafix/ScalafixModule.scala#L16

File: build.sc [hunk 1]
```diff
@@ -11,15 +11,18 @@ import mill._
...
     extends CrossScalaModule
     with TpolecatModule
     with StyleModule
     with GitVersionedPublishModule
     with SwaggerModelGenerator {
 
-  override def scalacOptions = super.scalacOptions().filter(_ != "-Wunused:imports")
+  override def scalacOptions = super.scalacOptions()
+    .filter(_ != "-Wunused:imports")
+    .filter(_ != "-Xfatal-warnings")
```
Comment (novakov-alexey-zz) [inline build.sc:24]: I could not compile it without disabling "warning as errors" flag. If I switch the project to 2.13.5, then there we get warning that some Circe Encoder/Decoder codecs are unused, however they are really used. I am not sure who is checking for unused implicits, is it scalac or scalafix.

Comment (novakov-alexey-zz) [reply build.sc:24]: This is an example of unused implicits: https://github.com/joan38/kubernetes-client/pull/90/checks?check_run_id=2904560216

**Comment by joan38** [comment]
Thanks!

**Comment by novakov-alexey-zz** [comment]
@joan38 welcome. You might want to check why Scala 2.13.5. makes some Circe codecs as unused, 
so that we could turn on the "-Xfatal-warnings" flag again. Currently, it is off, which makes code base a bit loose without it.

**Pull request merged**